index.html

```diff
@@ -139,6 +142,21 @@ <h2>Methods</h2>
         </dd>
       </dl>
     </section>
+    <section>
+      <h2>Modifications to existing procedures</h2>
+
+      <p>Append the following two paragraphs to the description of
+      <a data-cite="WEBRTC#dom-peerconnection-addicecandidate">addIceCandidate</a>:</p>
+
+      <p>If <a data-cite="WEBAPPSEC-CSP#should-block-rtc-connection">
+        should-block-rtc-connection</a> on the current global object returns
+      "Blocked", all candidates MUST be <a data-cite="WEBRTC#dfn-administratively-prohibited">
+        administratively prohibited</a>.</p>
+
+      <p>If a candidate is <a data-cite="WEBRTC#dfn-administratively-prohibited">,
+        the UA MUST NOT provide the candidate to the
+        <a data-cite="WEBRTC#dfn-ice-agent">ICE Agent</a>.</p>
```

Comment:
Once https://github.com/w3c/webrtc-pc/pull/2708 is merged (assuming it is merged as currently written), we can remove this, but I suppose it doesn't hurt to leave it here until then.

Reply:
Since that pr is merged, I've backed this out.